eng/versioning.targets

```diff
@@ -35,7 +35,7 @@
 
   <Target Name="AddUnsupportedOSPlatformAttribute" BeforeTargets="GenerateAssemblyInfo" AfterTargets="PrepareForBuild">
     <ItemGroup>
-      <AssemblyAttribute Include="System.Runtime.Versioning.UnsupportedOSPlatform" Condition="'@(_unsupportedOSPlatforms)' != ''">
+      <AssemblyAttribute Include="System.Runtime.Versioning.UnsupportedOSPlatform" Condition="'@(_unsupportedOSPlatforms)' != '' and '$(IsTestProject)' != 'true'">
```

Comment:
Most likely we don't want to add the `UnsupportedOSPlatform` attribute for test projects, let me know if am wrong

Reply:
Could we also move this condition to the target itself?

Reply:
Jep that condition makes more sense on the target.

Reply:
Yep, moved